app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_request
+      def create
+        @suggestion = BookSuggestion.new(suggest_params)
+        @suggestion.user_id = current_user.id if current_user
+
+        if @suggestion.valid?
```

Comment:
you can do @suggestion.save directly, if it's invalid it wouldn't save anything

Reply:
Understood. Notice that I'll need to render an error if the request is invalid, so I still need to call for validation